test/lib.rb

```diff
@@ -11,3 +11,22 @@
 require 'oppen'
 
 Minitest::Reporters.use!
+
+module PrettyPrinterTestsHelper
+  def self.split_string_into_tokens(string)
+    words = string.split.map do |word|
+      Oppen::Token::String.new(word)
+    end
+    result = [Oppen::Token::Break.new] * ((words.length * 2) - 1)
+    index = 0
+    words.each do |word|
+      result[index] = word
+      index += 2
+    end
+    [
+      Oppen::Token::Begin.new,
+      *result,
+      Oppen::Token::End.new,
+    ]
+  end
+end
```

Comment:
I don't think there's any need to put it in a separate module.
Let's reduce noise, make the token helper avaiable as is.

The better way to do it is to extend Ruby's `String` class itself, maybe with a `tokens` methods, so now we can do "this is my string".tokens.